js/components/Tab.jsx

```diff
@@ -0,0 +1,39 @@
+var React = require("react");
+
+var Tab = function(props) {
```

Comment:
Indeed it's temporary, it's just there to signal the state of the steps (not tabs anymore for that reason) and will be restyled later properly.